### PR TITLE
160 ODS component about accessibility - Add scrollview to see all the content

### PR DESCRIPTION
**Pull request by ludovic35**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Types of change

- Bug fix (non-breaking which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Refactoring (non-breaking change)
- Breaking change (fix or feature that would change existing functionality)

#### Accessibility

- [ ] My change follows accessibility good practices

#### Design

- [ ] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
- [ ] My change is compatible with responsive display

#### Documentation

- [ ] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [x] Manually test on simulators and on a physical device (Android & iOS, orientation change, dark/light mode)
- [x] Code review
- [ ] Design review
- [x] A11y review

### CODE REVIEW

File: app/CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Add a scroll view to see all the content in about screen ([#149](https://github.com/Orange-OpenSource/ods-flutter/issues/160))
```
Comment (paulinea) [inline app/CHANGELOG.md:20]: This is not the good issue number. `#149` should be replaced by `#160`

File: app/lib/ui/about/about_screen.dart [hunk 2]
```diff
@@ -35,27 +32,25 @@ class _AboutScreenState extends State<AboutScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
         body: SafeArea(
+            child: SingleChildScrollView(
+      child: Column(
+        children: [
+          Image(image: AssetImage('assets/il_about.png')),
```
Comment (paulinea) [inline app/lib/ui/about/about_screen.dart:38]: I think we can use these lines to allow the image to take all the space available when device is in landscape:
```dart
          Image(
            width: double.infinity,
            image: AssetImage('assets/il_about.png'),
            fit: BoxFit.cover,
          ),
```

**Review by paulinea — CHANGES_REQUESTED** [review]

File: app/CHANGELOG.md (hunk 1 above)
Comment (ludovic35) [reply app/CHANGELOG.md:20]: oups

File: app/lib/ui/about/about_screen.dart [hunk 3]
```diff
@@ -35,7 +35,7 @@ class _AboutScreenState extends State<AboutScreen> {
             child: SingleChildScrollView(
       child: Column(
         children: [
-          Image(image: AssetImage('assets/il_about.png')),
+          Image(image: AssetImage('assets/il_about.png'), fit: BoxFit.cover),
```
Comment (paulinea) [inline app/lib/ui/about/about_screen.dart:38]: The `width: double.infinity` is needed to fit full width

**Review by paulinea — CHANGES_REQUESTED** [review]

File: app/lib/ui/about/about_screen.dart (hunk 3 above)
Comment (ludovic35) [reply app/lib/ui/about/about_screen.dart]: Done

**Pull request merged**